### DISCUSSION TITLE
Chromium with MathML support

### DISCUSSION THREAD

**Discussion by iyutr** [Eloston/ungoogled-chromium#1526] [Ideas]
Hello developer!
could you please consider creating an Ungoogled Chromium build with includes from Igalia Chromium (https://github.com/Igalia)?
This is the only source code of Chromium with MathML support. MathML is necessary for people with a scientific mind.

**Comment by networkException** [comment]
Hello,

I don't belive that adding additional features like this is still in scope of this patchset as it wouldn't retain the chromium experience as closely as possible. According to a [blog article](https://mathml.igalia.com/news/2021/02/15/mathml-plans-for-2021/#new) they published a few months ago, MathML is actually being formalised as a web spec meaning that all browsers will eventually support it.

**Comment by iyutr** [comment]
Agreed, let's close the issue.

**Reply by networkException** [reply]
GitHub Discussions doesn't really have a way to close a discussion, if this would be in the Q&A category I could mark my comment as an answe but here it can just stay like this
